interface-definitions/containers.xml.in

```diff
@@ -64,6 +74,28 @@
               </leafNode>
             </children>
           </tagNode>
+          <leafNode name="publish-port">
```

Comment:
See above volume comment - the same applies here.

Reply:
Do I need to replace it to tag node? It will be difficult to handle such format data.
We can publish ports: 
- p 8888:88
- p 8282:53/udp
- p 127.0.0.1/8383:53/udp
```
vyos@r6-roll# set container name foo publish-port 
Possible completions:
   <xxxx:xx>    Host port xxxx to container port xx
   <xxxx:xx/udp>
                Host udp port xxxx to container port xx
   <xxxx-yyyy:xxxx-yyyy/udp>
                Host udp ports range xxxx-yyyy to container ports xxxx-yyyy
   <x.x.x.x/xxxx:xx>
                Host ip x.x.x.x port xxxx to container port xx

```

Reply:
done
```
set container name foo port map1 destination '22'
set container name foo port map1 protocol 'tcp'
set container name foo port map1 source '2222'
```